src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -546,14 +544,13 @@ def run_single(
                         )
 
                     # Also try to verify CI provenance contents.
-                    if verify_provenance:
-                        verified = []
-                        for ci_info in analyze_ctx.dynamic_data["ci_services"]:
-                            verified.append(verify_ci_provenance(analyze_ctx, ci_info, temp_dir))
-                            if not verified:
-                                break
-                        if verified and all(verified):
-                            provenance_l3_verified = True
+                    verified = []
+                    for ci_info in analyze_ctx.dynamic_data["ci_services"]:
+                        verified.append(verify_ci_provenance(analyze_ctx, ci_info, temp_dir))
+                        if not verified:
```

Comment:
I have a small comment on this part, this might not be relevant to this PR because you only remove the `if verify_provenance` condition and keep the body intact. 
`verify_ci_provenance` returns a bool, therefore `verified` at line 550 is always not empty. I wonder in which case is `verified` empty at line 550 ?

Reply:
It seems this should have been changed a while ago. I will update it so it checks the most recently added `bool` result and `breaks` based on that.